pkgs/flox-interpreter/default.nix

```diff
@@ -133,10 +134,10 @@ runCommandNoCC "flox-interpreter"
     ${shfmt}/bin/shfmt --diff $build_executable_wrapper
     rm $build_executable_wrapper/.editorconfig
 
-    # Create helper symlinks from `sbin/{sh,bash}` for use wrapping
-    # executables in build-manifest.nix. Put them in `/sbin` to avoid
-    # collisions with other packages that may provide these files.
-    mkdir -p $build_executable_wrapper/sbin
-    ln -s ${bash}/bin/bash $build_executable_wrapper/sbin/bash
-    ln -s ${bash}/bin/bash $build_executable_wrapper/sbin/sh
+    # Finally, leave a breadcrumb to document the nixpkgs URL used to build
+    # this version of the interpreter. This, together with the URL found
+    # within an environment's `manifest.toml` will determine the [potentially]
+    # two nixpkgs URLs used to build a given package.
+    echo "github:flox/nixpkgs/${inputs.nixpkgs.rev}?narHash=${inputs.nixpkgs.narHash}" > $out/nixpkgs-url
```

Comment:
Suggestion 1: 
We already derive COMMON_NIXPKGS_URL somewhere in a similar way, I think we should reuse that instead of copying that definition here

Suggestion 2:
consider `echo -n` so we don't need to strip out new lines at the other end.

Suggestion 3 (NB)
Put flox "support files" into a subdir instead of the env root (similar to .nix-support)

Reply:
Uploaded commit e26920b6405d590f5a37b290f38c7cab3fe0861b for item 3 in the list above. I like that better - thanks for the suggestion!